src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -10,127 +10,278 @@ redirects:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/-5--8DZynFE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+  </Collapser>
+  
+  <Collapser
+    id="streaming-method"
+    title="Streaming method"
+  >
+  In general, we recommend using the **event flow** streaming method. This is best for data that comes into your system frequently and steadily. There're specific cases where event timer might be a better method to choose but for your first alert, we recommend our default, **event flow**. To better understand which streaming method to choose, we recommend watching this brief video.  
+
+  <Video
+  type="wistia"
+  id="n6nei987ln"
 />
 
-<figcaption>
-  When you create a new policy, you can group your violations by policy, condition, or by violation (this creates an incident for each violation). These [incident preferences](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents/) settings determine how frequently you're notified when there's a violation.
-</figcaption>
+  </Collapser>
+
+    <Collapser
+    id="delay"
+    title="Delay"
+  >
+
+  Setting the delay is essentially the same thing as creating an aggregation buffer. Since the alerts system of New Relic only evaluates streaming data, you need to add a delay before any incidents will open on that condition. This means that if you set X as your aggregation window, for the first X minutes, no incidents will trigger alerts.
+
+  As an example, imagine that you set up an alert condition using a 1-hour aggregation window and a 1-minute sliding aggregation window. When you save the condition, the first hour will be spent building an aggregation buffer of streamed data. No violations will trigger alerts in this first hour. After that point, the condition will behave normally. Every minute, it will have a fresh 60m of data to aggregate and evaluate.
+
+  </Collapser>
+
+  <Collapser
+    id="gap-filling-strategy"
+    title="Gap-filling strategy"
+  >
+  We're creating an alert condition to notify our team of any latency issues with web transaction time. In this case, our application consistently sends New Relic data. There is a constant stream of signals being sent from our application to New Relic. In this case, there is no expected gap in signal so we won't need to select a gap-filling strategy. For this use case, and for your first alert, we recommend leaving the **gap-filling strategy** set to **none**. 
+
+  If you've a more inconsistent data set that sends signals to New Relic once every twenty-four hours, then, we'd recommend customizing this feature based on your teams specific needs. 
```

Comment:
I know we always try to use contractions, but you've always read a little weird to me. Again just a small personal thing I thought I'd bring up for consideration :)

Reply:
Yes! I agree. It kind of makes it sound like something out of Austen!